sample/build.gradle

```diff
@@ -122,7 +122,7 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
     implementation 'com.google.firebase:firebase-messaging:20.0.0'
     implementation 'com.google.firebase:firebase-core:17.2.0'
-
+    implementation 'com.google.android.material:material:1.1.0-alpha09'
```

Comment:
Why it's needed?

Reply:
I have used `BottomSheetDialog` of android material lib in `ChannelMoreActionDialog`